src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigAligner.java

```diff
@@ -0,0 +1,272 @@
+                    final byte[] sequence = contigInfo._2.toString().getBytes();
+                    final AlnRgn[] alnRgns = bwaAlignSequence(bwaMem, contigId, sequence);
+
+                    // todo: parse cigar for internal indels
+                    if (alnRgns.length > 1) {
+
+                        // filter out secondary alignments, convert to AlignmentRegion objects and sort by alignment start pos
+                        final List<AlignmentRegion> alignmentRegionList = Arrays.stream(alnRgns)
+                                .filter(a -> a.getSecondary() == -1)
+                                .map(AlignmentRegion::new)
+                                .sorted(Comparator.comparing(a -> a.startInContig))
+                                .collect(Collectors.toList());
+
+                        AlignmentRegion current = alignmentRegionList.get(0);
+                        AlignmentRegion previous;
+                        for (int i = 0; i < alignmentRegionList.size(); i++) {
+                            if (i > 0) {
+                                previous = current;
+                                current = alignmentRegionList.get(i);
+                                final AssembledBreakpoint assembledBreakpoint = new AssembledBreakpoint();
+                                assembledBreakpoint.contigId = contigId;
+
+                                assembledBreakpoint.region1 = previous;
+                                assembledBreakpoint.region2 = current;
+
+                                assembledBreakpoint.homology = "NA";
+                                if (assembledBreakpoint.region1.endInContig >= assembledBreakpoint.region2.startInContig) {
+                                    assembledBreakpoint.homology = getSequence(contigsCollection, contigId, assembledBreakpoint.region2.startInContig - 1, assembledBreakpoint.region1.endInContig);
+                                }
+
+                                assembledBreakpoint.insertedSequence = "NA";
+                                if (assembledBreakpoint.region1.endInContig < assembledBreakpoint.region2.startInContig - 1) {
+                                    assembledBreakpoint.insertedSequence = getSequence(contigsCollection, contigId, assembledBreakpoint.region1.endInContig, assembledBreakpoint.region2.startInContig);
+                                }
+
+                                assembledBreakpoints.add(assembledBreakpoint);
+                            }
+                        }
+                    }
+                }
+        } catch (final IOException e) {
+            throw new GATKException("could not execute BWA");
+        }
+
+        return assembledBreakpoints;
+    }
+
+    /**
+     * Pulls the sequence of the requested contig between the start and end coordinates from the contigsCollection
+     * @param contigsCollection
+     * @param contigId
+     * @param startInContig
+     * @param endInContig
+     * @return
```

Comment:
I think our convention is to either document fully, or omit tags altogether.  The `@param` and `@return` tags are just noise, I think.